src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -417,6 +421,17 @@ private void LoadLogs(ConsoleLogsSubscription newConsoleLogsSubscription)
 
             try
             {
+                var ignoreLogsBefore = _ignoreAllLogsBefore; // 'Global' ignore before timestamp, defaults to DateTime.Min
```

Comment:
Why does this have a default value if it's immediately set afterwards in the TryGetValue?